Dockerfile

```diff
@@ -30,36 +35,37 @@ ENV LDFLAGS=-s
 
 FROM base AS cpu
 # amd64 uses gcc which requires devtoolset-11 for AVX extensions while arm64 uses clang
-RUN if [ "$(uname -m)" = "x86_64" ]; then yum install -y devtoolset-11-gcc devtoolset-11-gcc-c++; fi
-ENV PATH=/opt/rh/devtoolset-11/root/usr/bin:$PATH
+RUN if [ "$(uname -m)" = "x86_64" ]; then dnf install -y gcc-toolset-11-gcc gcc-toolset-11-gcc-c++; fi
```

Comment:
Can probably remove the conditional since amd64 and arm64 base stages can use gcc-11